### ISSUE TITLE
List on contracts on front page

### ISSUE THREAD

**Issue by ethernomad** [DigixGlobal/doxity#8]
The front page looks a bit bare and the user has to click on Contracts to get to the APIs.

Could the contracts be listed on the front page?

**Comment by hitchcott** [comment]
Agreed, currently its populated purely by the README file, but I think you're right we can do better.

Maybe we just scrap the front page and have a 'readme' tab at the top of the contracts list.

I'll take a look at this shortly.
